include/gz/physics/GetRayIntersection.hh

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (C) 2024 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef GZ_PHYSICS_GETRAYINTERSECTION_HH_
+#define GZ_PHYSICS_GETRAYINTERSECTION_HH_
+
+#include <vector>
```

Comment:
where are you using `vector` ? Remove if this is not used

Reply:
In fact, `vector` is not used here.

Removed in https://github.com/gazebosim/gz-physics/pull/641/commits/967f43383f6e45c732d6d45737293be5bda940f1